lighthouse-core/audits/accessibility/aria-command-name.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Ensures every ARIA button, link and menuitem element has an accessible name
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+const AxeAudit = require('./axe-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if important elements have an accessible name. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  title: '`button`, `link` and `menuitem` elements have an accessible name',
+  /** Title of an accesibility audit that evaluates if important elements do not have accessible names. This title is descriptive of the failing state and is shown to users when there is a failure that needs to be addressed. */
+  failureTitle: '`button`, `link` and `menuitem` elements do not have an accessible name.',
+  /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'When an element doesn\'t have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn more](https://web.dev/aria-toggle-field-name/).',
```

Comment:
bummer that we made the web.dev docs for the audits *too* specific and helpful 😆 

do we have an issue or something in web.dev to track the new docs? I feel like all these new audits could get easily missed if they're already associated with a web.dev link

Reply:
https://github.com/GoogleChrome/web.dev/pull/4337